new-lamassu-admin/src/components/inputs/formik/Checkbox.js

```diff
@@ -5,7 +5,21 @@ import { Checkbox } from '../base'
 const CheckboxInput = memo(({ label, textAlign, fullWidth, ...props }) => {
   const { name, onChange, value } = props.field
 
-  return <Checkbox name={name} onChange={onChange} value={value} {...props} />
+  const settings = {
+    enabled: props.enabled ?? true,
+    label: label,
+    disabledMessage: props.disabledMessage ?? ''
+  }
```

Comment:
Since this is overriding the settings that comes with props, I think we need to consider all the fields here. It's missing at least the 'requirement' field.

Reply:
This is correct as well. The requirement field is something that I've added to blockcypher but it's not needed in this Checkbox component.